pkg/builder/webhook.go

```diff
@@ -32,10 +33,12 @@ import (
 
 // WebhookBuilder builds a Webhook.
 type WebhookBuilder struct {
-	apiType runtime.Object
-	gvk     schema.GroupVersionKind
-	mgr     manager.Manager
-	config  *rest.Config
+	apiType       runtime.Object
+	withDefaulter admission.CustomDefaulter
+	withValidator admission.CustomValidator
```

Comment:
```suggestion
	customDefaulter admission.CustomDefaulter
	customValidator admission.CustomValidator
```
nit (also fine for me to keep it as is, but compared to the other fields it seems consistent to me to drop the with prefix)